### DISCUSSION TITLE
Swift Package Manager instead of Cocoapods in Ios

### DISCUSSION THREAD

**Discussion by TohidMahmamoudvand** [codemagic-ci-cd/codemagic-docs#2477] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

iOS (Swift, Objective-C)

### Steps to reproduce

Can we use your CI-CD for an ios project that uses Swift Package Manager instead of Cocoapods?

### Expected results

-

### Actual results

-

**Comment by himesh-cm** [comment]
Hi @TohidMahmamoudvand, Yes, you can use SPM with codemagic. Is there any specific issue you are facing?
It will automatically handle the installation of packages specified in your `Package.swift` file.
Did you give it a go?

**Comment by himesh-cm** [comment]
While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!
